upgrade.py

```diff
@@ -666,6 +681,12 @@ def buildRestoreList(self, src_base):
 
         return restore_list
 
+        # Keep IPv6 enablement/disablement upon upgrades
+        self.restore_list += ['etc/sysctl.d/91-net-ipv6.conf']
+
+        # Keep user multipath configuration
+        self.restore_list += [{'dir': 'etc/multipath/conf.d', 're': re.compile(r'custom.*\.conf')}]
+
```

Comment:
Wow, @glehmann somehow we missed this mismerge

Reply:
Indeed `git range-diff` ignores merge commits and does not give the option to include them.

Reply:
we'd best rebase the ipv6 branch on 10.10.25+ instead to reduce conflicts